tests/requirements-devel.in

```diff
@@ -0,0 +1,4 @@
+-r requirements.in
+
+pygments >= 2.10.0
+sphinx-intl
```

Comment:
Why is sphinx-intl present here, but not in requirements.in? It was present in docs/docsite/requirements.txt, so it should better be in requirements.in as well (and be omitted here).

Reply:
@felixfontein Please bear with me as I'm probably making a meal of this. What I'm trying to achieve is restoring the same pattern we had with the two sets of requirements before the new docs repo.

I'm sure you recall. `docs/docsite/requirements.txt` were in use on the "devel" branch and `test/sanity/code-smell/docs-build-requirements.txt` were in use on "stable-*" branches.

My reason for putting `sphinx-intl` in `requirements-devel.in` is that the package was present in `docs/docsite/requirements.txt` only. If we move that package to `requirements.in` would that not effectively be the same as moving a package from `docs/docsite/requirements` to `test/sanity/code-smell/docs-build-requirements.txt`?

Reply:
Will follow up with https://github.com/ansible/ansible-documentation/issues/321